x-pack/platform/plugins/shared/fleet/public/applications/fleet/sections/agents/agent_details_page/components/agent_details/agent_details_overview.tsx

```diff
@@ -142,7 +144,15 @@ export const AgentDetailsOverviewSection: React.FunctionComponent<{
                 defaultMessage: 'Last activity',
               }),
               description: agent.last_checkin ? (
-                <FormattedRelative value={new Date(agent.last_checkin)} />
+                <EuiCopy textToCopy={agent.last_checkin}>
+                  {(copy) => (
+                    <span>
+                      {agent.last_checkin}
```

Comment:
We should use i18n helper to formatted date and time here see [here](https://github.com/elastic/kibana/blob/main/x-pack/platform/plugins/shared/fleet/public/applications/fleet/sections/agents/agent_list_page/components/agent_activity_flyout/helpers.tsx#L105-L112) for example